helpers/calendar/analytics_helper.py

```diff
@@ -94,48 +94,57 @@ def get_calendar_id_name(self, query):
                 raise JsonError(Message='No rooms in this location')
             else:
                 raise GraphQLError("No rooms in this location")
-        result = [{'name': room.name, 'calendar_id': room.calendar_id}
-                  for room in rooms_in_locations.all()]
+        result = [{
+            'name': room.name,
+            'room_id': room.id,
+            'calendar_id': room.calendar_id
+        }for room in rooms_in_locations.all()]
         return result
 
-    def get_all_events_in_a_room(self, calendar_id, min_limit, max_limit):
+    def get_all_events_in_a_room(self,
+                                 room_id,
+                                 event_start_time,
+                                 event_end_time):
         """ Get all events in a room
          :params
-            - calendar_id - for specific room
-            - min_limit, max_limit(Time range)
+            - room_id - for specific room
+            - event_start_time, event_end_time(Time range)
         """
-        try:
-            events_result = get_google_calendar_events(
-                calendarId=calendar_id, timeMin=min_limit,
-                timeMax=max_limit, singleEvents=True,
-                orderBy='startTime')
-        except Exception:
-            raise GraphQLError("Resource not found")
-        all_events = events_result.get('items', [])
-        return all_events
+        events = EventsModel.query.all()
+        result = []
```

Comment:
If this is a list of items, then the name should be plural. What does result refer to in this case?

Reply:
let me refactor this

Reply:
So in this case, the result meant the outcome of the query.  let me make this more readable

Reply:
what is the outcome of the query?

Reply:
room events